cluster-bootstrap/src/test/scala/akka/management/cluster/bootstrap/ClusterBootstrapSettingsSpec.scala

```
@@ -14,20 +15,26 @@ class ClusterBootstrapSettingsSpec extends AbstractBootstrapSpec {
   "ClusterBootstrapSettings" should {
 
     "have the expected defaults " in {
-      val settings = ClusterBootstrapSettings(config)
+      val settings = ClusterBootstrapSettings(config, NoLogging)
       settings.newClusterEnabled should ===(true)
       settings.joinDecider.selfDerivedHost.isEmpty should ===(true)
     }
 
     "have the expected overrides " in {
       val overrides = ConfigFactory.parseString("akka.management.cluster.bootstrap.new-cluster-enabled=off")
-      val settings = ClusterBootstrapSettings(overrides.withFallback(config))
+      val settings = ClusterBootstrapSettings(overrides.withFallback(config), NoLogging)
       settings.newClusterEnabled should ===(false)
     }
 
     "detect if an old property is set and fail fast, directing users to the new property" in {
```

update test description: fail fast

---

Doh